### DISCUSSION TITLE
Pronouns not showing up publicly

### DISCUSSION THREAD

**Discussion by Kaympe20** [community/community#142186] [Other Feature Feedback, Questions, & Ideas]
I added my pronouns to GitHub and they do show up when signed in, but when I sign out or another person looks at my profile, they do not show up.
Signed in:
![image](https://github.com/user-attachments/assets/a1883201-2e1f-4cbc-8d67-31681e976134)
Signed out:
![image](https://github.com/user-attachments/assets/23fc5f45-d78c-42f8-b34a-de61dc9f6530)

**Comment by spenserblack** [comment]
[The docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#adding-pronouns-to-your-profile) do mention this

> Your pronouns will only be visible to users that are signed in to GitHub.

But I have no idea why that would be the case :shrug:

**Comment by artyomboyko** [comment]
@Kaympe20 Good day. You can read about it [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#adding-pronouns-to-your-profile).
